src/main/java/org/jenkinsci/maven/plugins/hpi/AbstractJenkinsManifestMojo.java

```diff
@@ -213,7 +215,11 @@ protected void setAttributes(Manifest.ExistingSection mainSection) throws MojoEx
         addLicenseAttributesForManifest(mainSection);
         addPropertyAttributeIfNotNull(mainSection, "Plugin-ChangelogUrl", "hpi.pluginChangelogUrl");
         addPropertyAttributeIfNotNull(mainSection, "Plugin-LogoUrl", "hpi.pluginLogoUrl");
+
         addAttributeIfNotNull(mainSection, "Plugin-ScmUrl", getScmUrl());
+        addAttributeIfNotNull(mainSection, "Plugin-Scm-Connection", getSCMConnection());
```

Comment:
so that it could be obtained in the future without projects having to pollute their own pom.

jenkinsci/plugin-compat-tester#453